### ISSUE TITLE
EJT58 fails stitching with incorrect preallocated size

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/StitchIt#175]
The autoROI brain EJT58 fails to pre-allocate correctly:
`Y coordinate would place tile up to pixel value 3753 but image is pre-allocated up to 3480. SKIPPING TILE`

This is super annoying, as I thought I had fixed this.

**Comment by raacampbell** [comment]
Whilst looking into this I noticed that `determineStitchedImageExtent` reports a tile step size of zero for the last section, which is odd:

```
tileStep =

     0     0

269/269 11x1053 pixels. With far corner at pixel pos 3080/1382 (Inf by Inf tile step sizes).
```

This is because `tileStepSizeMM` comes out as zero.

**Comment by raacampbell** [comment]
Fixed -- 3312d941dc371cdbf361a2980ae34e051f55ec3b
